detection_rules/misc.py

```diff
@@ -249,6 +249,12 @@ def download_worker(rule_info):
     return kibana_rules
 
 
+@cached
```

Comment:
This was moved to avoid a circular reference, but ended up not being used. I left it because it is a needed change, but if it is a distraction, we can rip it and add separately